cmd/minikube/cmd/config/util.go

```diff
@@ -136,8 +137,31 @@ func EnableOrDisableAddon(name string, val string) error {
 	return enableOrDisableAddonInternal(addon, cmd, data, enable)
 }
 
+func isAddonAlreadySet(addon *assets.Addon, enable bool) error {
+
+	addonStatus, err := addon.IsEnabled()
+
+	if err != nil {
+		return errors.Wrap(err, "get the addon status")
+	}
+
+	if addonStatus && enable {
+		return fmt.Errorf("addon %s was already enabled", addon.GetAddon())
+	} else if !addonStatus && !enable {
+		return fmt.Errorf("addon %s was already disabled", addon.GetAddon())
+	}
+
+	return nil
+}
+
 func enableOrDisableAddonInternal(addon *assets.Addon, cmd bootstrapper.CommandRunner, data interface{}, enable bool) error {
 	var err error
+	// check addon status before enabling/disabling it
+	if err := isAddonAlreadySet(addon, enable); err != nil {
+		console.ErrStyle(console.Conflict, "%v", err)
+		os.Exit(exit.Unavailable)
```

Comment:
lets exit with 0, because we don't want to fail automated minikubes tests used CI, if their addon was already set, this will make their script to fail.

I rather we just warn the user but not exit with error. lets exit with code 0

Reply:
Done. Thanks Medya